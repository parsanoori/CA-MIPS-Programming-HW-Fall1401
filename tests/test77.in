60
258575362
671958579
285910742
249782433
665564767
482636728
534305647
195578780
678436926
427022010
339996139
1006454703
386528170
875158680
249334241
318456717
504814126
584763989
1056767536
857317578
184077297
545436492
885033230
918525019
81609075
1052069631
189161456
153856133
941982163
372065007
933747605
26941721
679784238
791715583
32234483
567734361
479996349
411067039
440555072
508708550
748688173
429622133
230507066
737722304
830422288
316734747
380954986
601423623
98884669
757591526
235464263
704303685
407208962
606902489
136718366
1010800181
845402874
733037454
235188424
661174496