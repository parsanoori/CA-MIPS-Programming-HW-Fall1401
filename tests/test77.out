26941721
32234483
81609075
98884669
136718366
153856133
184077297
189161456
195578780
230507066
235188424
235464263
249334241
249782433
258575362
285910742
316734747
318456717
339996139
372065007
380954986
386528170
407208962
411067039
427022010
429622133
440555072
479996349
482636728
504814126
508708550
534305647
545436492
567734361
584763989
601423623
606902489
661174496
665564767
671958579
678436926
679784238
704303685
733037454
737722304
748688173
757591526
791715583
830422288
845402874
857317578
875158680
885033230
918525019
933747605
941982163
1006454703
1010800181
1052069631
1056767536